Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Primitives/SwitchPresenter.bind

```diff
@@ -0,0 +1,79 @@
+    <ComboBox x:Name="Lookup" Header="Look up reservation" SelectedIndex="0"
+              Margin="0,0,0,8">
+      <x:String>Select an option</x:String>
+      <x:String>Confirmation Code</x:String>
+      <x:String>E-ticket number</x:String>
+      <x:String>Mileage Plan number</x:String>
+    </ComboBox>
+    <!-- SwitchPresenter binds to a value -->
+    <controls:SwitchPresenter Value="{Binding SelectedItem, ElementName=Lookup}">
+      <!-- And then only dynamically displays the Case with the matching Value -->
+      <controls:Case Value="Confirmation Code">
+        <StackPanel>
+          <TextBox Name="ConfirmationCodeValidator"
+                   ui:TextBoxExtensions.Regex="^[a-zA-Z]{6}$"
+                   Header="Confirmation code"
+                   PlaceholderText="6 letters" />
+          <TextBlock Visibility="{Binding (ui:TextBoxExtensions.IsValid), ElementName=ConfirmationCodeValidator}">Thanks for entering a valid code!</TextBlock>
+        </StackPanel>
+      </controls:Case>
+      <controls:Case Value="E-ticket number">
+        <StackPanel>
+          <TextBox Name="TicketValidator"
+                   ui:TextBoxExtensions.Regex="(^\d{10}$)|(^\d{13}$)"
+                   Header="E-ticket number"
+                   PlaceholderText="10 or 13 numbers" />
+          <TextBlock Visibility="{Binding (ui:TextBoxExtensions.IsValid), ElementName=TicketValidator}">Thanks for entering a valid code!</TextBlock>
+        </StackPanel>
+      </controls:Case>
+      <controls:Case Value="Mileage Plan number">
+        <TextBox Name="PlanValidator"
+                 Header="Mileage Plan #"
+                 PlaceholderText="Mileage Plan #" />
+      </controls:Case>
+      <!-- You can also provide a default case if no match is found -->
+      <controls:Case IsDefault="True">
+        <TextBlock>Please select a way to lookup your reservation above...</TextBlock>
+      </controls:Case>
+    </controls:SwitchPresenter>
+
+    <Border Height="2" Background="Gray" Margin="0,16"/>
+
+    <!-- Scenario using an Enum -->
+    <ComboBox x:Name="AnimalPicker"
+              Header="Pick an Animal"
+              ItemsSource="{ui:EnumValues Type=enums:Animal}"
+              SelectedIndex="0"/>
+    <controls:SwitchPresenter Value="{Binding SelectedItem, ElementName=AnimalPicker}"
+                              TargetType="enums:Animal"
+                              Padding="16">
+      <controls:Case Value="Cat">
+        <TextBlock FontSize="32">🐈</TextBlock>
+      </controls:Case>
+      <controls:Case Value="Dog">
+        <TextBlock FontSize="32">🐕</TextBlock>
+      </controls:Case>
+      <controls:Case Value="Bunny">
+        <TextBlock FontSize="32">🐇</TextBlock>
+      </controls:Case>
+      <controls:Case Value="Parrot">
+        <TextBlock FontSize="32">🦜</TextBlock>
+      </controls:Case>
+      <controls:Case Value="Squirrel">
+        <TextBlock FontSize="32">🐿</TextBlock>
+      </controls:Case>
```

Comment:
No LLama here?

Reply:
I thought @Sergio0694 had added one originally as this is from another sample, so I was surprised there wasn't one too... I suppose we can add one... 😋